<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>credit</class>
 <widget class="QWidget" name="credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_res">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>100</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 1px solid gray;
   background-color: white;
   border-radius: 5px;
}
QPushButton:pressed {
    background-color: rgb(240, 240, 240);
}</string>
   </property>
   <property name="text">
    <string>рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>175</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
  border-radius: 5px;
  background-color : white;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>175</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
  border-radius: 5px;
  background-color : white;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>175</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
  border-radius: 5px;
  background-color : white;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>181</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
  border-radius: 5px;
  background-color : rgb(225, 225, 225);
}
</string>
      </property>
      <property name="text">
       <string>сумма кредита (руб)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
  border-radius: 5px;
  background-color : rgb(225, 225, 225);
}</string>
      </property>
      <property name="text">
       <string>срок кредита (месяцы)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
  border-radius: 5px;
  background-color : rgb(225, 225, 225);
}</string>
      </property>
      <property name="text">
       <string>процентная ставка (%)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>181</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
  border-radius: 5px;
  background-color : rgb(225, 225, 225);
}</string>
      </property>
      <property name="text">
       <string>ежемесячный платеж</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
  border-radius: 5px;
  background-color : rgb(225, 225, 225);
}</string>
      </property>
      <property name="text">
       <string>начисленные проценты</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
  border-radius: 5px;
  background-color : rgb(225, 225, 225);
}</string>
      </property>
      <property name="text">
       <string>долг + процентs</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSpinBox" name="months">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>60</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sum">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>100.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="rate">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>179</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
  border-radius: 5px;
  background-color : rgb(225, 225, 225);
}</string>
   </property>
   <property name="text">
    <string>тип платежей</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_an">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>185</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
  qproperty-alignment: 'AlignVCenter | AlignRight';
}</string>
   </property>
   <property name="text">
    <string>аннуитетные</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_dif">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>185</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
  qproperty-alignment: 'AlignVCenter | AlignRight';
}</string>
   </property>
   <property name="text">
    <string>дифференцированные </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
